### ISSUE TITLE
Create ongoing training and learning opportunities for users

### ISSUE THREAD

**Issue by choldgraf** [2i2c-org/docs#128]
### Background and proposal

**Context**

Many users do not immediately know how to best-use 2i2c's infrastructure for their research / educational use-case. We cannot cover domain-specific training for all communities, but we could provide a some regular touch-points to help onboard users and clear up problems with major workflows. This would also be a good opportunity to get some face-to-face feedback about the service itself.

**Proposal**

We should set up a recurring seminar or workshop series that covers major questions and confusions that users may have.

### Implementation guide and constraints

**Topics to cover**

This wouldn't be unique to a scientific field, but might cover the major use-cases we expect 95% of our users to have. For example:

- Teaching with nbgitpuller
- Approaches to auto-grading notebooks
- Reading / writing data from the cloud
- Basic usage of Dask
- Scaling with Dask Gateway

**Where to store material**

We could store each of these as tutorials on docs.2i2c.org, in a dedicated "user tutorials" section (and perhaps another set of tutorials in a "hub administrator tutorials" section). The documentation would be step-by-step, and if we were running live workshops to step others through an analysis, these tutorials would be reference material we'd use.

### Updates and ongoing work

- This is relevant to the LEAP hub, as this community requires some ongoing training for their community, ref: https://github.com/2i2c-org/infrastructure/issues/1050

**Comment by damianavila** [comment]
> The documentation would be step-by-step, and if we were running live workshops to step others through an analysis, these tutorials would be reference material we'd use.

Makes sense to me... wondering if we should focus on videos or written stuff first? 🤔

**Comment by choldgraf** [comment]
@damianavila what I've found useful is to start off writing a "Tutorials" section in docs, and then in a workshop, use that tutorial ~verbatim as a step-by-step guide. That will surface issues, confusion points, etc and those can be fed back into the tutorial docs.

It is trickier if you want to maintain "slides" and things like this in addition to the tutorial docs. One option there is to define slides that are lightweight and just point to the tutorial to step through. That way there's one source of truth.

**Comment by damianavila** [comment]
> Or, we could do something clever and use a RISE presentation or something

Interesting 😜 

> but I suspect we'll need to wait until somebody has more dedicated time for this

 I agree.
